### ISSUE TITLE
Not in the NPM registry

### ISSUE THREAD

**Issue by theorye** [claylo/loopback-connector-dynamodb#3]
NT

**Comment by gyss** [comment]
Still not found. 404

**Comment by claylo** [comment]
@gyss @theorye I've largely fallen out of love with Loopback. You're welcome to roll your own connector for this functionality. Any DynamoDB work I do going forward will not be loopback-specific.

**Comment by theorye** [comment]
Thanks.  Would you be willing to share  why you've fallen out of love?

**Comment by claylo** [comment]
@theorye Sure. I felt particularly burned by this issue: https://github.com/strongloop/loopback-component-oauth2/issues/27

I believed (and still believe) there are several ways they could have conducted themselves which would have been more above board. Given the amount of time I've spent digging through their source, reporting on issues or submitting patches, I have no interest in continuing to support something that does the license switcheroo. I dropped Ext.js/Sencha over this in the past, and am now dropping Loopback as well.

There are plenty of other projects out there to contribute to that don't behave the way they did in this instance.
